go/iot/protocol/configure.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if [ "$#" -ne 3 ]; then
+  echo "Illegal number of parameters"
+  echo "First is the hostname, second is the ip address, third is the ssh password"
+  exit -1
+fi
+
+go get -u github.com/hybridgroup/gobot
+
+go get -u contrib.go.opencensus.io/exporter/stackdriver
+
+go get -u go get -u go.opencensus.io
+
+sudo apt-get install sshpass
+
+cd $(go env GOPATH)/src/github.com/census-ecosystem/oepncensus-experiments/go/iot/protocol
+
+sudo apt-get install gcc-arm-linux-gnueabihf
+
+chmod u+x ./run.sh
+
```

Comment:
Do you need to init and configure Cloud credentials in this script?

Reply:
solved.